meerkat_abacus/config.py

```diff
@@ -116,7 +116,8 @@ def __init__(self):
         elif self.stream_data_source == "AWS_S3":
             self.get_data_from_s3 = 1
             self.s3_data_stream_interval = os.environ.get("S3_DATA_STREAM_INTERVAL", 3600)
-
+        elif self.stream_data_source == "NO_STREAMING":
+            pass  # Don't do any streaming.
```

Comment:
This comment doesn't add anything useful neither it indicates a block of code :)

Reply:
I personally disagree.  Python pass statements are null statements often synonymous with an unfinished code stub.  I think a comment is valuable here as it makes it clear the pass statement is an intentional part of the developer's design.  The first article I google about pass statements begins their second para with "A good way to use pass is to hold the place of code that isn't ready or hasn't been written yet." (https://www.pythoncentral.io/using-pythons-pass-statement/). I want to explicitly avoid any possibility for this to be an interpretation of my code. 

However, I am not the maintainer for this project so it is for you, @Gulfa and @jsoppela to decide.

Reply:
I second Jonathan, it adds readability and cost no money.